test/integration/bootstrap.spec.js

```diff
@@ -111,7 +111,7 @@ describe('bootstrap()', () => {
       routes: [{
         steps: {}
       }]
-    })).should.Throw(`Cannot find views at ${root}/invalid_path`)
+    })).should.Throw('ENOENT: no such file or directory, scandir')
```

Comment:
I think is related to this https://github.com/UKHomeOfficeForms/hof/pull/296/files#diff-cb055c8e3497e3772fc444a77c4aed8483453e7580ffe7ddb223bdffaff43c57L10

HOF still complains so that is the main thing and it tells you where it is too